### PR TITLE
=build update headers to cStyleComment

### PR DESCRIPTION
**Pull request by He-Pin**

### CODE REVIEW

File: project/CopyrightHeader.scala [hunk 1]
```diff
@@ -67,11 +65,35 @@ trait CopyrightHeader extends AutoPlugin {
...
 
+    def parseStartAndEndYear(header:String):Option[(String,Option[String])] = header match {
+      case CopyrightHeaderPattern(years, null, _)     =>
+        Some((years,None))
+      case CopyrightHeaderPattern(years, endYears, _) =>
+        Some((years,Some(endYears)))
+      case _                                          =>
+        None
+    }
+
```
Comment (hepin1989) [inline project/CopyrightHeader.scala]: Add this to parse the existing headers information

File: project/CopyrightHeader.scala [hunk 2]
```diff
@@ -67,11 +65,35 @@ trait CopyrightHeader extends AutoPlugin {
...
     override def apply(text: String, existingText: Option[String]): String = {
```
Comment (hepin1989) [inline project/CopyrightHeader.scala]: was updated to recreate headers with 
```
/*
*
*/
```
style

**Review by patriknw — APPROVED** [review]
LGTM, thanks

**Comment by hepin1989** [comment]
I have rebased

**Comment by johanandren** [comment]
I'm missing some context here, not having them as Javadoc/Scaladoc seems the right thing, but is there some further motivation to change them?

**Comment by hepin1989** [comment]
@johanandren The current PR is a blocker of enabling the Java formatter plugin

**Review by johanandren — APPROVED** [review]
Ah, I see. thanks for clarifying. LGTM.

**Pull request merged**